tests/behaviour/cube.feature

```diff
@@ -187,12 +246,47 @@ Feature: Cube!
-          {'csv_column_title': 'Value', 'structural_definition': {'data_type': 'decimal', 'unit': None}, 'csv_column_uri_template': None, 'uri_safe_identifier_override': None}, 
-          {'csv_column_title': 'Measure', 'structural_definition': {'measures': [{'label': 'Length', 'description': None, 'parent_measure_uri': None, 'source_uri': None, 'uri_safe_identifier_override': None, 'arbitrary_rdf': []}]}, 'csv_column_uri_template': None, 'uri_safe_identifier_override': None}, 
-          {'csv_column_title': 'Unit', 'structural_definition': {'units': [{'label': 'cm', 'description': None, 'source_uri': None, 'uri_safe_identifier_override': None, 'arbitrary_rdf': [], 'base_unit': None, 'base_unit_scaling_factor': None, 'qudt_quantity_kind_uri': None, 'si_base_unit_conversion_multiplier': None}]}, 'csv_column_uri_template': None, 'uri_safe_identifier_override': None}
-      ]
+    [
+      "Dimension",
+      "Value",
+      "Measure",
+      "Unit"
+    ]
+    """
+    Then The cube data should match
+    """
+    [
+      {
+        "Dimension":"a",
+        "Value":0,
+        "Measure":"length",
+        "Unit":"cm"
+      },
+      {
+        "Dimension":"b",
+        "Value":1,
+        "Measure":"length",
+        "Unit":"cm"
+      },
+      {
+        "Dimension":"c",
+        "Value":2,
+        "Measure":"length",
+        "Unit":"cm"
+      },
+      {
+        "Dimension":"d",
+        "Value":3,
+        "Measure":"length",
+        "Unit":"cm"
+      },
+      {
+        "Dimension":"e",
+        "Value":4,
+        "Measure":"length",
+        "Unit":"cm"
+      }
+    ]
     """
 
   Scenario: Output a cube when an inline code list is defined using code list config schema v1.0, and the sort order is defined with sort object
```

Comment:
Whilst I appreciate that the tests are less brittle now than they were, some of the tests like these don't really test much outside of the build not failing any more. For instance, I think this test should be asserting that the sort order of the code list items has been correctly set.

Reply:
@robons The sort order is tested in the following unit tests at `test_code_list_config.py`: `test_code_list_config_model`, `test_code_list_config_model_without_schema`, `test_code_list_config_model_with_hierarchy`.